frappe/public/css/desk.css

```diff
@@ -467,7 +467,7 @@ fieldset[disabled] .form-control {
 }
 .modal-backdrop {
   opacity: 0.5;
-  position: fixed;
+  position: absolute;
```

Comment:
Please edit the desk.less file

Reply:
Hi @netchampfaris 
I also changed the .less file since only changing less file and not .css file dint do anything
Is this right?
I´m not familiar with .less but if it is for dynamic styling why didnt it do anything after bench build and clear website cache?
I thought less is either to build a .css or to be used standalone... however according to chrome console the desk.css is the one controling the item modal-backdrop.
please clarify